config/config.json

```diff
@@ -110,7 +110,10 @@
         "PasswordResetSalt": "vZ4DcKyVVRlKHHJpexcuXzojkE5PZ5eL",
         "SendPushNotifications": false,
         "PushNotificationServer": "",
-        "PushNotificationContents": "generic"
+        "PushNotificationContents": "generic",
+        "EnableEmailBatching": false,
+        "EmailBatchingBufferSize": 256,
+        "EmailBatchingInterval": 30
```

Comment:
If a user where to set this to 1. It might be possible for multiple go routines to be in `checkPendingNotifications` at the same time because there is no lock.